src/components/FinalPoem.js

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './FinalPoem.css';
 
 const FinalPoem = (props) => {
 
+  const [finalSentence, setFinalSentence] = useState('')
```

Comment:
Does `finalSentence` need to be in state?  Could props work just as well?

Also the tests make some assuptions about the props that you've chanced, which is why they don't pass.